cfgov/unprocessed/css/molecules/inset.less

```diff
@@ -7,9 +7,16 @@
         <div class="o-full-width-text-group">
             <div class="m-full-width-text"></div>
             <div class="m-inset">
-                <div class="m-quote">
+                <h2 class="header-slug">
+                    <span class="header-slug_inner"></span>
+                </h2>
+                <div class="m-inset_image-container">
+                    <div class="m-inset_image"></div>
+                    <img class="m-inset_image" />
                 </div>
+                <div class="m-quote"></div>
             </div>
+            <div class="m-inset_text"></div>
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/pull/2678/commits/74c81849f5ecd2cb7d3e9cc07c71d8dc53489f31.